LLDBDebugger/LLDBProtocol/LLDBConnector.h

```diff
@@ -70,10 +72,10 @@ class LLDBConnector : public wxEvtHandler
 
 protected:
     clSocketClient::Ptr_t m_socket;
-    LLDBNetworkListenerThread* m_thread;
+    std::unique_ptr<LLDBNetworkListenerThread> m_thread;
```

Comment:
I made this a unique_ptr so Cleanup() was safe(r) to call multiple times, but then made the mod in LLDBPlugin::OnLLDBCrashed() to SetGoingDown() so it's not really needed I guess. But seems nicer :smile: